lib/shared/errors.js

```diff
@@ -228,7 +234,7 @@ exports.getDescription = (error) => {
   }
 
   if (error.stack) {
```

Comment:
Why should that be the case? Rather than let it carrying on with the rest of the code below?